src/transform/collecting/qualitative_measurement_collector.rs

```diff
@@ -0,0 +1,64 @@
+use crate::config::context::{Context, ContextKind};
+use crate::extract::ContextualizedDataFrame;
+use crate::extract::contextualized_dataframe_filters::Filter;
+use crate::transform::PhenopacketBuilder;
+use crate::transform::collecting::traits::Collect;
+use crate::transform::error::CollectorError;
+
+#[allow(dead_code)]
+#[derive(Debug)]
+pub struct QualitativeMeasurementCollector;
+
+impl Collect for QualitativeMeasurementCollector {
+    fn collect(
+        &self,
+        builder: &mut PhenopacketBuilder,
+        patient_cdf: &ContextualizedDataFrame,
+        phenopacket_id: &str,
+    ) -> Result<(), CollectorError> {
+        let qualitative_measurement_scs = patient_cdf
+            .filter_series_context()
+            .where_data_context_kind(Filter::Is(&ContextKind::QualitativeMeasurement))
+            .collect();
+
+        for qual_measurement_sc in qualitative_measurement_scs {
+            let (loinc_id, unit_ontology_prefix) = qual_measurement_sc
+                .get_data_context()
+                .try_as_qualitative_measurement()
```

Comment:
Just for my understanding.

This function will try to convert a context enum into its components, right?
And will only be an Ok(), if the selected try function alignes with the enum variant.